### PR TITLE
fix(sass): changed slashes to math.div

### PR DESCRIPTION
**Pull request by peter7cole**
## Description
This PR addresses a Deprecation Warning that we've noticed in our builds for the Webex admin portal.
Linked announcement here: https://sass-lang.com/documentation/breaking-changes/slash-div

## Related Issue
[Deprecation Warning: Sass Slashes #1176](https://github.com/momentum-design/momentum-ui/issues/1176)

## Motivation and Context
We would like to clear our warnings

## Screenshots:
Screen Shot 2022-03-10 at 9 34 21 PM

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: core/scss/components/loader/placeholders.scss [hunk 1]
```diff
@@ -133,12 +133,7 @@
           }
 
           .md-spinner-progress__mask {
-            clip: rect(
-              0px,
-              #{$circle-size},
-              #{$circle-size},
-              calc(#{$circle-size} / 2)
-            );
+            clip: rect(0px, #{$circle-size}, #{$circle-size}, calc(#{$circle-size} / 2));
```
Comment (peter7cole) [inline core/scss/components/loader/placeholders.scss:136]: having some issues with the formatting, but this matches the `printWidth: 100` in the .prettierrc.json file

File: core/scss/settings/settings.scss [hunk 2]
```diff
@@ -4244,10 +4241,10 @@ $definition-list-margin-bottom: 0.5rem;
 
 // ex.: golden ratio
 // current: 16 * 1.71429 / 4 = 6.85716
-$base-scale: $base-unit-rem * $base-line-height-ratio / $base-unit-px;
+$base-scale: $base-unit-rem * calc($base-line-height-ratio / $base-unit-px);
```
Comment (eazy-g) [inline core/scss/settings/settings.scss:4244]: Tests are failing on these...I think you have to use `#{$base-line-height-ratio}` format when referencing variables in `calc()`

**Review by eazy-g — COMMENTED** [review]
Thanks for taking this on @peter7cole ! I was just running into same headache

**Comment by peter7cole** [comment]
> Thanks for taking this on @peter7cole ! I was just running into same headache

@eazy-g thank you, I'm going to link a new PR and close this one as I'm running into a bit of a headache geting github to sync formatting

File: core/scss/settings/settings.scss (hunk 2 above)
Comment (peter7cole) [reply core/scss/settings/settings.scss:4244]: got it, thank you for the catch

**Pull request closed without merging**